lib/cli/src/__tests__/commands/check/utils/check-schema-compatibility.test.ts

```diff
@@ -236,6 +236,32 @@ describe("Schema Compatibility Checks", () => {
     );
   });
 
+  it("should check the compatibility of additionalProperties schemas", () => {
```

Comment:
The previous version of `check-schema-compatibility.ts` also missed checking for compatibility of the `additionalProperties` schemas if both the base spec and implementation spec allowed additional properties